evm-contracts/src/v0.6/interfaces/AggregatorInterface.sol

```diff
@@ -1,29 +1,12 @@
 pragma solidity >=0.6.0;
 
-import "./HistoricAggregatorInterface.sol";
+interface AggregatorInterface {
```

Comment:
I'm getting a bit confused by the naming of the interfaces. Isn't this an AggregatorV2Interface?

Reply:
Can this be out of scope for this PR?

Reply:
Ok

Reply:
I prefer leaving it as is, because then consumers right now don't have to change the paths they import things from. Alternatively, we could copy the file to AggregatorV2.sol, but I don't think we should remove the existing interface.